profiles/milkshakegum.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/29112173/190592868-c5f1d4a3-5d60-48d3-8e60-f28dce110ba4.png",
+    "issueId": 66,
+    "name": "Milk",
+    "username": "milkshakegum"
+  }
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient